### ISSUE TITLE
Fails when using proxyfmu_booter

### ISSUE THREAD

**Issue by ljamt** [open-simulation-platform/proxy-fmu#57]
Starting `proxyfmu_booter`
```
.\proxyfmu_booter.exe --port 9090

Press any key to quit...
[proxyfmu] Found proxyfmu executable: "proxyfmu.exe"
[proxyfmu] Booting FMU instance 'OSOM'..
[proxyfmu] FMU instance 'OSOM' instantiated using port 65510
[proxyfmu] External proxy process for instance 'OSOM' returned with status -1073741819
```
Using OspSystemStructure.xml:
```
<OspSystemStructure xmlns="http://opensimulationplatform.com/MSMI/OSPSystemStructure" version="0.1">
        <StartTime>0.0</StartTime>
        <BaseStepSize>0.05</BaseStepSize>
        <Algorithm>fixedStep</Algorithm>
        <Simulators>
                <Simulator name="OSOM" source="proxyfmu://localhost:9090?file=OSOM.fmu" stepSize="0.05"/>
        </Simulators>
</OspSystemStructure>
```
Fails when running `cosim`

```
 .\cosim.exe run .\OspSystemStructure.xml
error: No more data to read.
```

It works fine when not using the booter by altering to the following:
`<Simulator name="OSOM" source="proxyfmu://localhost?file=OSOM.fmu" stepSize="0.05"/>`

**Comment by markaren** [comment]
@mrindar had a similar issue with cosim v.0.5.1. The issue is weird, because the booter works with cosim 0.5.0, but not inline proxyfmu. With 0.5.1, inline proxyfmu works, but not booter..

**Comment by markaren** [comment]
I also have no issues when running libcosim master with booter and inline proxyfmu.

**Comment by mrindar** [comment]
Let me summarise the two situations:

*A*
  - .\proxyfmu_booter.exe --port 9090
  - `.\cosim.exe run .\OspSystemStructure.xml` 
  - `source="proxyfmu://localhost:9090?file=OSOM.fmu"` 
  - CRASHES

*B*
  - `.\cosim.exe run .\OspSystemStructure.xml` 
  - `source="proxyfmu://localhost?file=OSOM.fmu"` 
  - WORKS

If I understand the system correctly, the only difference here is that in situation *A*, `proxyfmu_booter.exe` starts `proxyfmu.exe`, and in situation *B* `cosim.exe` starts `proxyfmu.exe`. Is there some fundamental difference between the two in how they start `proxyfmu.exe`?

Alternatively, is there some difference between `proxyfmu_booter.exe` and `cosim.exe` in how they link libraries, or which compiler flags are used?

**Comment by markaren** [comment]
Fixed by #58
